authentication/account/service/user.go

```diff
@@ -208,6 +208,8 @@ func (s *userServiceImpl) notifyIdentityBeforeDeactivation(ctx context.Context,
 	if err := s.ExecuteInTransaction(func() error {
 		notificationDate := now()
 		identity.DeactivationNotification = &notificationDate
+		scheduledDeactivation := identity.LastActive.Add(s.config.GetUserDeactivationInactivityPeriodDays())
```

Comment:
actually, it should be something like `now() + s.config.GetUserDeactivationInactivityPeriodDays() - s.config.GetUserNotificationDeactivationInactivityPeriodDays()`, to make sure that the deactivation happens 7 days (by default) after the notification is sent, even if the last active date was 2 months ago. Otherwise, we may set a deactivation date to somewhere in the past.

Reply:
I've updated the code in accordance.